src/components/avatar/Avatar.jsx

```diff
@@ -8,13 +8,11 @@ const DEFAULT_COLOR = ICON_COLOR.GRAY_SECONDARY;
 const BORDER_SIZE = 2;
 
 export const SIZE = {
-  XSMALL: {className: 'xsmall', iconSize: 16},
   SMALL: {className: 'small', iconSize: 24},
   NORMAL: {className: 'normal', iconSize: 32},
   LARGE: {className: 'large', iconSize: 48},
   XLARGE: {className: 'xlarge', iconSize: 64},
-  XXLARGE: {className: 'xxlarge', iconSize: 96},
-  XXXLARGE: {className: 'xxxlarge', iconSize: 120}
+  XXLARGE: {className: 'xxxlarge', iconSize: 120}
```

Comment:
When I will bump SG in brainly-frontend I will update places where this size is used, there is no appearance of Avatar size XXLARGE, so I will just change XXXLARGE (one place, change avatar page) to XXLARGE.